compiler/next/include/chpl/queries/ErrorMessage.h

```diff
@@ -21,6 +21,7 @@
 #define CHPL_QUERIES_ERRORMESSAGE_H
 
 #include "chpl/queries/Location.h"
+#include "chpl/util/iteration.h"
```

Comment:
Why is this include here? Did you mean to add a method to ErrorMessage or something?

Reply:
The include is here so I can add an a way to iterate over the errors.

Reply:
Lol, wrong file